doc/toolbox-enter.1.md

```diff
@@ -32,6 +33,13 @@ Enter a toolbox container with the given NAME. This is useful when there are
 multiple toolbox containers created from the same base image, or entirely
 customized containers created from custom-built base images.
 
+Either use this option **or** argument <*container*> (the argument <*container*>
```

Comment:
I have split this out in https://github.com/containers/toolbox/pull/681

I think this text can be simplified by simply not mentioning the older --container option. The more we mention something, the more we encourage people to actually use it. Since we consider the older option deprecated, let's just not mention it.

What we should also do is try to emit a deprecation warning from the code when someone does actually use it.